projects/kit/components/input-files/input-files.directive.ts

```diff
@@ -43,8 +43,10 @@ export class TuiInputFilesDirective {
     @HostBinding('attr.capture')
     get capture(): TuiInputFilesOptions['capture'] {
         return (
-            this.el.nativeElement.getAttribute('capture') as TuiInputFilesOptions['capture'] | null
-        ) ?? this.options.capture;
+            (this.el.nativeElement.getAttribute('capture') as
```

Comment:
I would just change `get capture(): string | null` and drop all that nonsense :D

Reply:
https://github.com/Tinkoff/taiga-ui/pull/4221